### DISCUSSION TITLE
Add a new apps discovery page/tab

### DISCUSSION THREAD

**Discussion by f-x1-1** [ImranR98/Obtainium#1910] [Ideas]
A dedicated tab or page for discovering apps would be a fantastic addition to Obtanium, allowing users to instantly find and install new applications.

**Comment by DwainZwerg** [comment]
Obtainium does not want to be an app store, it wants to be a simple updater. Obtainium does not want to promote any app, does not want to suggest any app. The [ConfigurationStore](https://apps.obtainium.imranr.dev/) is also only intended for complicated configurations, not for proposing apps. Whether an app/source is considered useful/trustworthy should be decided by the responsible and experienced users themselves. Your suggestion goes against the aims of Obtainium.
